packages/react/src/PricingOptions/PricingOptions.tsx

```diff
@@ -252,18 +262,47 @@ const PricingOptionsPrice = forwardRef<HTMLParagraphElement, PricingOptionsPrice
         weight="normal"
         {...rest}
       >
-        <Text as="span" className={styles['PricingOptions__price-currency-symbol']} size="500" weight="normal">
+        <Text
+          as="span"
+          className={styles['PricingOptions__price-currency-symbol']}
+          font="hubot-sans"
+          size="500"
+          weight="normal"
+        >
           {currencySymbol}
         </Text>
 
-        <Text as="span" className={styles['PricingOptions__price-value']} size="700" weight="normal">
+        <Text as="span" className={styles['PricingOptions__price-value']} font="hubot-sans" size="700" weight="normal">
           {children}
         </Text>
 
-        <Text as="span" className={styles['PricingOptions__price-currency-code']} size="500" weight="normal">
+        <Text
+          as="span"
+          className={styles['PricingOptions__price-currency-code']}
+          font="hubot-sans"
```

Comment:
Curious if the use of Hubot Sans is just for Pricing Options, Universe or how we generally want to represent numbers in PB @danielguillan?